tests/ignite/metrics/test_rouge.py

```diff
@@ -0,0 +1,296 @@
+import os
+
+import nltk
```

Comment:
Useless ?

Reply:
we need to nltk to download punkt which is needed by py-rouge